src/styles/stars.scss

```diff
@@ -0,0 +1,19 @@
+@import './variables';
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  gap: 4px;
+
+  &__star {
+    width: $star-size;
+    height: $star-size;
+    background-image: $star-image-default;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &--4 .stars__star:nth-child(-n + 4) {
```

Comment:
Consider adding all classes with modifiers to keep them working

Reply:
I do not understand why to do this, if it works and so, because it increases the code